src/ImageSharp/Processing/Processors/Transforms/Resize/ResizeKernel.cs

```diff
@@ -66,21 +70,87 @@ public Vector4 Convolve(Span<Vector4> rowSpan)
         [MethodImpl(InliningOptions.ShortMethod)]
         public Vector4 ConvolveCore(ref Vector4 rowStartRef)
         {
-            ref float horizontalValues = ref Unsafe.AsRef<float>(this.bufferPtr);
+#if SUPPORTS_RUNTIME_INTRINSICS
+            if (Fma.IsSupported)
+            {
+                float* bufferStart = this.bufferPtr;
+                float* bufferEnd = bufferStart + (this.Length & ~3);
+                Vector256<float> result256_0 = Vector256<float>.Zero;
+                Vector256<float> result256_1 = Vector256<float>.Zero;
+                var mask = Vector256.Create(0, 0, 0, 0, 1, 1, 1, 1);
```

Comment:
Should be `Vector.Load` with a ROS.

Reply:
The codegen in this very specific case seems to actually be just a `vxorps` (which is actually... weird), so I'm looking into this. It seems to be ok even though we're not using a `ROS` though, or actually better than that too.
Will update in a bit 🙂